hub/core/query/query.py

```diff
@@ -28,7 +28,11 @@ def __init__(
             for tensor in dataset.tensors.keys()
             if normalize_query_tensors(tensor) in query
         ]
-        self._blocks = expand(dataset, self._tensors)
+        try:
+            self._blocks = expand(dataset, self._tensors)
+        except Exception:
```

Comment:
Why not except specific exceptions?

Reply:
we should catch specific exceptions, so that we dont ignore any other errors unknowingly.